src/python/pants/option/global_options.py

```diff
@@ -110,6 +110,9 @@ def register_bootstrap_options(cls, register):
     register('-q', '--quiet', type=bool, recursive=True, daemon=False,
              help='Squelches most console output. NOTE: Some tasks default to behaving quietly: '
                   'inverting this option supports making them noisier than they would be otherwise.')
+    register('--show-rust-3rdparty-logs', type=bool, default=False, advanced=True,
```

Comment:
Annoying naming nit (sorry), but: renaming this to be be more namespaced (ie `--log-show-rust-3rdparty`) would make the help more scannable.